scripts/obtain/dotnet-install.ps1

```diff
@@ -151,18 +151,19 @@ function Invoke-With-Retry([ScriptBlock]$ScriptBlock, [int]$MaxAttempts = 3, [in
 function Get-Machine-Architecture() {
     Say-Invocation $MyInvocation
 
-    # possible values: AMD64, IA64, x86
+    # possible values: amd64, x64, x86, arm64, arm
     return $ENV:PROCESSOR_ARCHITECTURE
 }
 
-# TODO: Architecture and CLIArchitecture should be unified
 function Get-CLIArchitecture-From-Architecture([string]$Architecture) {
     Say-Invocation $MyInvocation
 
     switch ($Architecture.ToLower()) {
         { $_ -eq "<auto>" } { return Get-CLIArchitecture-From-Architecture $(Get-Machine-Architecture) }
-        { ($_ -eq "amd64") -or ($_ -eq "x64") } { return "x64" }
-        { $_ -eq "x86" } { return "x86" }
+        { ($_ -like "amd64") -or ($_ -like "x64") } { return "x64" }
```

Comment:
One question from powershell-ignorant me: why `like` vs `eq`?

Reply:
"-like" is case-insensitive
It for when someone types "X86" instead of "x86" on the command-line

Reply:
Is it a whole-string match or a sub-string match?  I assume `eq` was whole.

Reply:
Erm, never mind.  I think I was looking at something else.  This isn't regex, just some wild character characters.  Assuming this isn't substring matching, 👍

Reply:
"-eq" is already case-insensitive in Powershell; my mistake